lib/commonConfig.js

```diff
@@ -160,7 +160,7 @@ function processEnvironmentVariables() {
         config.contextBroker.ngsiVersion = process.env.IOTA_CB_NGSI_VERSION;
     }
     if (process.env.IOTA_JSON_LD_CONTEXT){
-        config.contextBroker.jsonLdContext = process.env.IOTA_JSON_LD_CONTEXT.split(';');
+        config.contextBroker.jsonLdContext = process.env.IOTA_JSON_LD_CONTEXT.split(',');
```

Comment:
This should be a regex split including optional whitepace e.g. `/,\s*/`, as you just know someone will attempt to provision using:

- `IOTA_JSON_LD_CONTEXT=http:/context1.json-ld, http:/context2.json-ld,  http:/context3json-ld`

Instead of:

- `IOTA_JSON_LD_CONTEXT=http:/context1.json-ld,http:/context2.json-ld, http:/context3json-ld`

And then blame the code for not finding `%20http:/context2.json-ld`